### PR TITLE
chore: bump retina-shell base image

### PR DESCRIPTION
**Pull request by nddq**
# Description

Bump retina-shell base image to latest version
## Related Issue

If this pull request is related to any issue, please mention it here. Additionally, make sure that the issue is assigned to you before submitting this pull request.

## Checklist

- [ ] I have read the [contributing documentation](https://retina.sh/docs/contributing).
- [ ] I signed and signed-off the commits (`git commit -S -s ...`). See [this documentation](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification) on signing commits.
- [ ] I have correctly attributed the author(s) of the code.
- [ ] I have tested the changes locally.
- [ ] I have followed the project's style guidelines.
- [ ] I have updated the documentation, if necessary.
- [ ] I have added tests, if applicable.

## Screenshots (if applicable) or Testing Completed

Please add any relevant screenshots or GIFs to showcase the changes made.

## Additional Notes

Add any additional notes or context about the pull request here.

---

Please refer to the [CONTRIBUTING.md](../CONTRIBUTING.md) file for more information on how to contribute to this project.

### CODE REVIEW

File: shell/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
-# mcr.microsoft.com/azurelinux/base/core:3.0.20241005
-FROM mcr.microsoft.com/azurelinux/base/core@sha256:7ec490b605aac8a44aed0b0695b0ee6ae976ec898afd9ac8d5613d7f3ce2b07b
+# mcr.microsoft.com/azurelinux/base/core:3.0.20241101
```
Comment (MikeZappa87) [inline shell/Dockerfile:1]: What purpose is this line serving?

Comment (nddq) [reply shell/Dockerfile:1]: since we are pinning the images by sha256 tags, it might be helpful to indicate which of the actual image the tag belongs to

Comment (MikeZappa87) [reply shell/Dockerfile:1]: Thanks, makes sense

**Comment by nddq** [comment]
closed in favor of https://github.com/microsoft/retina/pull/1120

**Pull request closed without merging**